### DISCUSSION TITLE
Comparto mi repo para los que no han terminado su primera práctica Frontend

### DISCUSSION THREAD

**Discussion by khisus19** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#150] [Show and tell]
No es gran cosa por cuestión de tiempo y porque no tengo ni idea de como funcionan los bufetes de abogados pero quizá les sirva para los que se encuentren luchando para terminarlo.

Si quieren que les comparta el archivo del buyer persona o el público objetivo (los tengo en formato ai de Illustrator) me escriben. Si les gustó algo del repo no olviden dar estrellita.
`https://github.com/khisus19/LaunchX-PrimeraSemana`

**Comment by Burstniart** [comment]
Excelente presentación bro, muy buenos diseños te aventaste y la info esta bastante completa.

**Reply by pepeabel** [reply]
ja, la pagina no existe

**Reply by JairFraga** [reply]
hay que ponerle un "/" al final del URL y listo :)

**Reply by khisus19** [reply]
Gracias. Ya reparé el enlace

**Reply by GloriaCoralCerecedo** [reply]
No me aparece nada

**Reply by SergioPinto98** [reply]
La pagina ya no existe :(

**Reply by khisus19** [reply]
Tienen razón. No sé qué ocurre con el enlace que no redirige al repo. Para verlo solo hay que copiar la dirección manualmente en la barra de navegación.

**Comment by ahranza** [comment]
Hola! Todo lo hiciste en Illustrator?

**Reply by khisus19** [reply]
El buyer persona, el target audience, el ux y ui los hice en Illustrator porque es el programa que mejor manejo. Para escribir luego el código me parece más cómodo hacer los diseños desde figma porque desde ahí puedes ver los tamaños de todo y escribir el código.

**Reply by ahranza** [reply]
gracias! No le había entendido ya hasta ahorita que vi cómo lo hiciste.

**Comment by davidtoString** [comment]
Gracias, muy buena guia.

**Comment by NoraBecerra** [comment]
Excelente aportación, gracias por compartir, con esto me doy una idea de como se deberá hacer, ya coloqué mi estrellita.

**Comment by NoraBecerra** [comment]
Una pregunta, ¿cómo entregaste la información dentro del formulario ya que indica que insertes el link de los diferentes documentos y además del repositorio, y si bien veo que tienes todos los archivos en el repositorio, como agregaste los demás links?

**Reply by AlejandroPA76** [reply]
Creo que en ese apartado se debe entregar por ejemplo si tu buyer persona lo guardaste en tu repositorio en una carpeta, debes de enviar el link de la carpeta donde se encuentre, o en su caso si lo hiciste con "miro" o con otra herramienta online debes de colocar el link para que puedan verlo, y asi con cada una de las katas.

**Reply by khisus19** [reply]
Lo que hice fue subir cada punto a Google drive. Esos enlaces del drive fueron los que puse en el formulario. Es muy fácil trabajar con Google drive.
